packages/angular/src/app/pages/analytics-sales-report/analytics-sales-report.component.html

```diff
@@ -82,7 +82,7 @@
 
     <card-analytics titleText="Sales Performance" className="sales" [component]="chart">
       <div filter class="sales-filter" >
-      <dx-drop-down-button stylingMode="text" [useSelectMode]="true"
+      <dx-drop-down-button stylingMode="text" [useSelectMode]="true" [dropDownOptions]="{width: 65}"
```

Comment:
Set width: 'auto' and add this style:
`dx-dropdownbutton-content {
  .dx-list-item {
    display: block;
  }
}`
It is a bug of components